sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command_func.py

```diff
@@ -148,88 +148,80 @@ def command(
     """Creates a Command object which can be used inside a dsl.pipeline function or used as a standalone Command job.
 
     :param name: The name of the Command job or component.
-    :type name: str
-    :param description: The description of the Command.
-    :type description: str
-    :param tags: Tag dictionary. Tags can be added, removed, and updated.
-    :type tags: dict[str, str]
-    :param properties: The job property dictionary.
-    :type properties: dict[str, str]
-    :param display_name: The display name of the job.
-    :type display_name: str
-    :param command: The command to be executed.
-    :type command: str
-    :param experiment_name: The name of the experiment the job will be created under. Defaults to current directory
-        name.
-    :type experiment_name: str
+    :type name: Optional[str]
+    :param description: The description of the Command. Defaults to None.
+    :type description: Optional[str]
+    :param tags: Tag dictionary. Tags can be added, removed, and updated. Defaults to None.
+    :type tags: Optional[dict[str, str]]
+    :param properties: The job property dictionary. Defaults to None.
+    :type properties: Optional[dict[str, str]]
+    :param display_name: The display name of the job. Defaults to randomly generated name.
```

Comment:
Is the random name procedural in some way? Also grammar nit:

```suggestion
    :param display_name: The display name of the job. Defaults to a randomly generated name.
```

Reply:
It's a random adjective + random noun + random integer/char sequence for now. For example, a recent job I submitted has the display name of "mighty_giraffe_t1rdcf75y3". Since we don't control that code/functionality, I don't want to hardcode an expectation of what the pattern will be.